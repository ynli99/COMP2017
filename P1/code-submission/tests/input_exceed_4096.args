Sydney Melbourne "11:00:00"
