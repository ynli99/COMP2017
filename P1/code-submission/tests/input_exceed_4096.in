ThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacename::ThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacenameThisisaverylongplacename::13:00:00
Sydney::Melbourne::13:59:40
