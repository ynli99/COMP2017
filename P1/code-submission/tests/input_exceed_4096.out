The next train to Melbourne from Sydney departs at 13:59:40
